setup.py

```diff
@@ -52,7 +52,8 @@
             'ivadomed_convert_to_onnx=ivadomed.scripts.convert_to_onnx:main',
             'ivadomed_extract_small_dataset=ivadomed.scripts.extract_small_dataset:main',
             'ivadomed_download_data=ivadomed.scripts.download_data:main',
-            'ivadomed_training_curve=ivadomed.scripts.training_curve:main'
+            'ivadomed_training_curve=ivadomed.scripts.training_curve:main',
+            'ivadomed_n4_correction=ivadomed.scripts.n4_correction:main'
```

Comment:
Again, to be discussed a the meeting to ensure 1) whether we want to generalize the magnetic field inhomogeneity correction beyond N4, 2) we okay with adding additional CLI command like this for magnetic field inhomogeneity correction.

Reply:
can you pls add it to next meeting agenda?